### ISSUE TITLE
Re-enable Yaegi testing (maybe?)

### ISSUE THREAD

**Issue by jasongoodwin** [bluecatengineering/traefik-aws-plugin#9]
Yaegi testing is done when the plugin is released but I disabled it as it has no dependency support. You have to vendor plugins, but we have to then copy them all to keep the build stable in our repo:

```
Go modules are not supported yet. Until that, it is necessary to install the source into $GOPATH/src/github.com/traefik/yaegi to pass all the tests.
```

I had a really hard time finding examples that worked and without access to the repo, the cycle time was too slow as our build files don't run and they won't run on a fork.

A lot of plugins just test w/ go as the traefik plugin workflow will catch any issues before publishing.

**Comment by yoeluk** [comment]
I agree. We should introduce this back into the CI/CD to test more principally during Ci/CD. It is worth noting that _now_ traefik provides the pipeline to both build and release... see [here](https://github.com/traefik-plugins/workflows/tree/main/.github/workflows), and for example on how to use it see [this](https://github.com/traefik-plugins/traefik-jwt-plugin/blob/main/.github/workflows/build.yaml#L15).

**Comment by jasongoodwin** [comment]
Yeah you have to copy the vendored plugins into the go src and I just couldn't iterate on it easily enough without admin access to the project to get it to work. I was wasting a lot of time waiting for approval to run the pipeline. It should be as easy as restoring that to the workflow and just copying everything but I couldn't find an example that worked and couldn't get a pass.
